runtime/pkg/graceful/http.go

```diff
@@ -11,26 +11,37 @@ import (
 
 const httpShutdownTimeout = 15 * time.Second
 
+type Options struct {
```

Comment:
This package also has some non-serving related logic in the `context.go` file, so I would suggest renaming this type to `ServeOptions`